m4/ax_cxx_compile_stdcxx.m4

```diff
@@ -0,0 +1,982 @@
+# ===========================================================================
```

Comment:
Do you need to pull this in, or can the AC_PREREQ(2.59) line be raised to ensure this support is present?